MapboxSearch/sdk/src/androidTest/java/com/mapbox/search/OfflineSearchEngineIntegrationTest.kt

```diff
@@ -516,28 +516,6 @@ internal class OfflineSearchEngineIntegrationTest : BaseTest() {
         assertTrue(results.isEmpty())
     }
 
-    @Test
-    fun testConsecutiveRequests() {
```

Comment:
This is a flaky test, we don't have a stable way to make the first request quite slow that second request starts before completion of the first. There's a similar test on the Native SDK side so we can safely remove it from platfoms.